src/user_pref_model.py

```diff
@@ -15,12 +15,14 @@ class UserPrefModel:
     def __init__(self, database_name: pathlib.Path):
```

Comment:
so is the directory to the database being passed in, or a full path that includes the filename within that directory.
It looks like this variable name should be database_directory or database_directory_path